src/game_server/handlers/character.rs

```diff
@@ -299,7 +302,7 @@ impl BaseNpc {
                 head_customization_override: "".to_string(),
                 hair_customization_override: "".to_string(),
                 body_customization_override: "".to_string(),
-                override_terrain_model: false,
+                override_terrain_model: !self.use_terrain_model,
```

Comment:
```suggestion
                override_terrain_model: self.use_terrain_model,
```
I think the negation here is a typo, since we'd want to override when this is true